apps/prairielearn/src/components/AssessmentRegenerate.html.ts

```diff
@@ -28,12 +28,11 @@ export function RegenerateInstanceModal({ csrfToken }: { csrfToken: string }) {
 export function RegenerateInstanceAlert() {
   return html`
     <div class="alert alert-warning alert-dismissible fade show">
-      You are viewing your own assessment instance and have instructor permissions.
+      Instructor in student view:
```

Comment:
I think we could even just remove this entirely?
```suggestion
```

Reply:
I think originally there was some concern that instructors would mistakenly think that _students_ would see this option when completing an assessment, hence the very verbose message. I think there may still be some merit to that concern (though maybe this won't be an issue in practice).

Here's a potential compromise:

<img width="1318" alt="Screenshot 2024-09-24 at 11 23 48" src="https://github.com/user-attachments/assets/1ee70f61-f233-4bad-adff-b92a439d1439">

Reply:
Maybe just use:
```suggestion
      Instructor:
```
I think adding the "students will not see this message" may be too verbose...

Reply:
Maybe "Instructors can regenerate this assessment instance to pick up..."? The "Instructor: ..." reads strangely to me.

Reply:
@nwalters512 I think of it as salutation. Hey instructors: You can do this. I think it's concise without confusion.

Reply:
I like the "Instructors:" preamble.